docs/Docker.md

```diff
@@ -21,9 +21,48 @@ Run the following commands from the DIM cloned directory on your machine:
 * `ctrl+c` to stop
 * `docker-compose up -d` to start in detached mode
 * `docker-compose stop` to stop detached mode
-* `docker-compose build` to re-build if the compose files change
 
-### Tips and How-To's
+### Tips and Troubleshooting
 
-- when using docker for your development platform, you will need to install new node dependencies from within a running container. This can be done by
-running `docker-compose exec yarn install nameofdependency webpack`
+You can get an interactive terminal with `docker-compose exec webpack bash`.
+
+## Commit hook won't run
+
+If you `git commit` inside the container, you may see:
+
+    Can't find Husky, skipping pre-commit hook
+    You can reinstall it using 'npm install husky --save-dev' or delete this hook
+    Can't find Husky, skipping prepare-commit-msg hook
+    You can reinstall it using 'npm install husky --save-dev' or delete this hook
+
```

Comment:
I don't think we should recommend removing hooks, as they are where linting occur etc.

Reply:
I think that's outside our direct/easy control, it's [from the husky tool](https://github.com/typicode/husky/blob/4e1d440be7e3058036d24e7c5558c09448772af2/sh/husky.sh#L20).